Loop/src/loop.hxx

```diff
@@ -541,6 +541,7 @@ public:
   template <int CI, int CJ, int CK, int VS = 1, int N = 1, typename F>
   inline CCTK_ATTRIBUTE_ALWAYS_INLINE void
   loop_outermost_int(const vect<int, dim> &group_nghostzones,
+                     const vect<vect<bool, dim>, 2> &is_sym_bnd,
```

Comment:
I'd document what `is_sym_bnd[f][d]` stands for. Does this order (f then d) match what is used eg in `cctk_bbox` (see https://www.einsteintoolkit.org/usersguide/UsersGuide.html#x1-98000C1.6.2). Looking at the Cactus example it does seem to be different since the Cactus code uses:

```
An array of 2cctk_dim integers (in the order [dim_0^{min}, dim_0^{max}, dim_1^{min}, dim_1^{max}, ... ]), ...
```

Reply:
The `is_sym_bnd` array follows the layout of the `symmetry` array from `driver.hxx`, which seems to be arranged in a different order from what Cactus does:
https://github.com/EinsteinToolkit/CarpetX/blob/a941c336a422030a1d4d72edcc38f69446ee623e/CarpetX/src/driver.hxx#L211
So the patch here is consistent with CarpetX, but not Carpet, which might be confusing.